kolena/asset.py

```diff
@@ -188,6 +190,22 @@ def _data_type() -> _AssetType:
         return _AssetType.MESH
 
 
+@dataclass(frozen=True, config=ValidatorConfig)
+class DocumentAsset(Asset):
+    """
+    A document file in a cloud bucket or served at a URL.
+
+    Only `.txt` and `.pdf` file types are supported.
```

Comment:
Looks like we also support `.md` and `.log` files: https://github.com/kolenaIO/marina/blob/miller/kol-7802-hinge-health-bug-class-level-metrics-interaction-when/src/components/Studio/components/GalleryItem/Document/StudioDocument.tsx#L23